benchmarkList.md

```diff
@@ -106,11 +106,15 @@ DRB179-thread-sensitivity-yes.cc                    |Y6	    | Conflicting writes
 DRB180-miniAMR-yes.c                                |Y6	    | Race by shared index variable                                       												| MiniAMR app               
 DRB181-SmithWaterman-yes.c                          |Y6	    | Race appears with larger data size                                  												| Smith-Waterman app
 DRB183-atomic3_yes.c                                |Y3     | Races because the write is not protected by atomic                                          | [Sequential Consistency for Data-race-free Programs](https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)
-DRB185-bar1_yes.c                                   |Y3     | wrong 2-thread flag barrier using busy-waits, race [inline]                                               |[Sequential Consistency for Data-race-free Programs](https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)
-DRB187-bar2_yes.c                                   |Y3     | each thread uses its own lock, which doesn't work                                          |[Sequential Consistency for Data-race-free Programs](https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)
-DRB189-bar3_yes.c                                   |Y3     | one synchronization missed [inline]                                            |[Sequential Consistency for Data-race-free Programs](https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)
-DRB191-critsec2_yes.c                               |Y3     | has race due to different critical section names                                  |[Sequential Consistency for Data-race-free Programs](https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)      
-DRB193-critsec3_yes.c                               |Y3     | has race due to different critical section names                                   |[Sequential Consistency for Data-race-free Programs](https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)
+DRB185-bar1_yes.c                                   |Y3     | wrong 2-thread flag barrier using busy-waits, race [inline]                                 | [Sequential Consistency for Data-race-free Programs](https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)
```

Comment:
I just noticed that filenames should use hyphens, not underscores. Please correct this for all your files added. @LChenGit